tests/resources/create_repodata_input.json

```diff
@@ -0,0 +1,51 @@
+{
+    "FETCH": [
```

Comment:
Where is all of this data coming from? Is this populated data or hard coded at wont need to be changed?